site/book-theme/highlight.js

```diff
@@ -1,3 +1,10 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// SPDX-FileCopyrightText: 2023 'mdBook contributers' (https://github.com/rust-lang/mdBook)
+// SPDX-License-Identifier: MPL-2.0
+
 /*
   Highlight.js 10.1.1 (93fd0d73)
   License: BSD-3-Clause
```

Comment:
The MPL header might be incorrect for this file. Did the upstream project modify it?

Reply:
Done.